tests/holder/test_events.py

```diff
@@ -2,6 +2,8 @@
 
 # pylint: disable=redefined-outer-name
 
+import typing
+
```

Comment:
Was this addition necessary? I don't see `typing` used anywhere in the other additions to this test.